torchvision/models/googlenet.py

```diff
@@ -104,8 +74,8 @@ def __init__(self, num_classes=1000, aux_logits=True, transform_input=False, ini
             self.aux1 = inception_aux_block(512, num_classes)
             self.aux2 = inception_aux_block(528, num_classes)
         else:
-            self.aux1 = None
-            self.aux2 = None
+            self.aux1 = None  # type: ignore[assignment]
+            self.aux2 = None  # type: ignore[assignment]
```

Comment:
It would be better to add

```
self.aux1: Optional[InceptionAux]
self.aux2: Optional[InceptionAux]
```
above the `if` statement. Otherwise `mypy` now thinks that `None` is not a valid value of `self.aux1` and `self.aux2`.

To do that we either need to move the helper classes above `GoogLeNet` or quote them in the annotations.

Reply:
Between quotes and moving the helper class, which one do you think is best?

Reply:
Depends on https://github.com/pytorch/vision/pull/2857#discussion_r510329158. I would move the helper classes, but lets discuss this there.

Reply:
I prefer if we just annotate with quotes. This facilitates looking for "blame" etc.

Reply:
Done!